### ISSUE TITLE
Rent

### ISSUE THREAD

**Issue by ErinDeveloping** [renzuzu/renzu_motels#23]
Can you add a function to select how often you need to pay rent like every week day or month for example or if thats already a function?

**Comment by renzuzu** [comment]
i dont quite understand.

do you mean an option to enable in config if this motel is should be day,hour,month, when renting

**Comment by ErinDeveloping** [comment]
Yes I would love to have that you need to pay every week insted 

A config like Config.RentTime -- week month hour day 

For exampel

**Comment by renzuzu** [comment]
still need more testing.

my test using qbcore looks good.
